documentation/book/con-delegated-previliges.adoc

```diff
@@ -0,0 +1,13 @@
+[id='delegate-previliges-{context}']
+=  Delegated privileges
+
+When the Cluster Operator deploys resources for a desired `Kafka` resource it also creates `ServiceAccounts`, `RoleBindings` and `ClusterRoleBindings`, as follows:
+
+* The Kafka broker pods use a `ServiceAccount` called `<cluster-name>-kafka` (where `<cluster-name>` is a placeholder for the name of the `Kafka` resource)
```

Comment:
Make italics and remove the bit in parens - we explain this in the documentation conventions.